daq/host.py

```diff
@@ -176,9 +175,8 @@ def _get_test_config(self, test):
         return get_test_config(self._loaded_config, test)
 
     def _test_enabled(self, test):
-        fallback_config = {'enabled': test in self._CORE_TESTS}
-        test_config = self._get_test_config(test) or fallback_config
-        return test_config.get('enabled', True)
+        test_config = self._get_test_config(test) or {}
```

Comment:
what's the impetus behind this change?  The intent was to have a basic set of things that would do something even in the absence of a valid external configuration.

Reply:
This brings inconsistencies in which tests are enabled; some test(as in the core tests) will run when "enabled" is not in the config and some tests won't. I have found this core concept + host_tests + module_configs to bring extra confusions on which test are run ahead of time. With this, the module config needs to explicitly enable each test before it is run, so it is clearer on which tests are configured to run. 
DAQ should reject invalid configurations instead of assuming some configurations. That is how we approached other configs.